<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>127</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RowsLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="FillButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>127</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Columns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ColsLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>309</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_i">
       <property name="text">
        <string>i =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_i"/>
     </item>
     <item>
      <widget class="QLabel" name="label_j">
       <property name="text">
        <string>j =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_j"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="SetValueButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Set value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GetValueButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Get value</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>218</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>new value = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NewElementLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead_from_fole"/>
    <addaction name="actionWrite_to_file"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionFill"/>
    <addaction name="actionSet_Value"/>
    <addaction name="actionGet_value"/>
    <addaction name="actionClear_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTable"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_from_fole">
   <property name="text">
    <string>Read from file</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + R">Ctrl+R</string>
   </property>
  </action>
  <action name="actionWrite_to_file">
   <property name="text">
    <string>Write to file</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + W">Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + E">Ctrl+E</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + F">Ctrl+F</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSet_Value">
   <property name="text">
    <string>Set value</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + S">Ctrl+S</string>
   </property>
  </action>
  <action name="actionGet_value">
   <property name="text">
    <string>Get value</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + G">Ctrl+G</string>
   </property>
  </action>
  <action name="actionClear_2">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string extracomment="Ctrl + A">Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
